src/permitInfo/utils/getTokensFromTokenList.ts

```diff
@@ -3,7 +3,7 @@ import { BASE_PATH } from '../const.ts'
 import { Token } from '../types.ts'
 import { join } from 'node:path'
 
-export function getTokens(chainId: number, tokenListPath: string | undefined): Array<Token> {
+export function getTokensFromTokenList(chainId: number, tokenListPath: string | undefined): Array<Token> {
```

Comment:
@alfetopito , i believe this method should also filter the tokens by `chainId`. 

I feel unsure if you omitted this on porpoise to discover tokens that are not listed for that specific chain but share the same address as one that is listed

As im unsure, i will play safe here and will NOT change this behaviour. Please, consider doing the filter when you are back

Reply:
You are right, it could as well filter tokens here.
Not a big problem though as they are filtered out later in the script.
This was meant as a simple fn to load tokens from a token list, taking into account that Goerli as a list of its own.